roles/calico/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+calico_image: ciscocloud/calico-node
+calico_image_tag: v0.4.8-mi
+
+bintray_baseurl: "https://bintray.com/artifact/download/loshkovskyi/generic"
```

Comment:
just a placeholder here, but we'll want to change this to download from the CiscoCloud org before we merge this.